code/lab_0.py

```diff
@@ -0,0 +1 @@
+print('Hello World')
```

Comment:
Looks good but for future labs make sure they are in their section folder. Example: `code/python/lab01/lab01.py`